packages/opentelemetry-node/test/NodeTracerProvider.test.ts

```diff
@@ -154,8 +154,16 @@ describe('NodeTracerProvider', () => {
       assert.strictEqual(span.isRecording(), false);
     });
 
-    // @todo: implement
-    it('should start a Span with always sampling');
+    it('should start a Span with always sampling', () => {
```

Comment:
What happens if parent is not sampled?

Reply:
Added another test for this. The child should not be sampled